pkgs/http_profile/test/http_profile_response_data_test.dart

```diff
@@ -26,6 +26,48 @@ void main() {
     backingMap = profileBackingMaps.lastOrNull!;
   });
 
+  group('HttpProfileRedirectData', () {
+    test('equal', () {
+      expect(
+          HttpProfileRedirectData(
+              statusCode: 302, method: 'GET', location: 'http://somewhere'),
+          HttpProfileRedirectData(
+              statusCode: 302, method: 'GET', location: 'http://somewhere'));
+    });
```

Comment:
There is another test case below called "equal". I think you meant to delete this one, because the one below subsumes this one.

Reply:
I meant to call the one below "not equal", which I've done now. Would you prefer that I combine the two tests?

Reply:
Ah. No, this is good. I misread the case below and thought these exact same lines appeared again, but they don't.